src/controllers/UserController.js

```diff
@@ -0,0 +1,58 @@
+import UserService from '../services/UserService';
+import passwordHelper from '../helpers/resetPasswordEmail';
+import HashPassword from '../helpers/hashPassword';
+
+const { hashPassword } = HashPassword;
+const { getAUser } = UserService;
+
+/**
+ * @export
+ * @class Users
+ */
+class Users {
+  /**
+   * @static
+   * @param {Object} req request object
+   * @param {Object} res response object
+   * @returns {Object} response
+   */
+  static async forgotPasswordController(req, res) {
+    const { email } = req.body;
+
+    const user = await getAUser({ email });
+    if (!user) {
+      return res.status(404).json({
+        status: 404,
+        message: 'User not found',
+      });
+    }
+
+    passwordHelper.resetPasswordEmailHelper(req, user);
+    return res.status(201).json({
+      status: 201,
+      message: 'A reset link has been sent to your email. Please check your email!',
+    });
+  }
+
+  /**
+   * @static
+   * @param {Object} req request object
+   * @param {Object} res response object
+   * @returns {Object} response
+   */
+  static async resetPasswordController(req, res) {
+    const { password } = req.body;
+    const { id } = req.user;
+    const user = await getAUser({ id });
+    const hashedPassword = hashPassword(password);
+    user.password = hashedPassword;
+    user.save();
+    passwordHelper.resetPasswordSuccessfulHelper(user);
+    return res.status(200).json({
+      status: 200,
+      message: 'Password reset successfull!',
```

Comment:
Good work @nziokaivy!
Would you mind customizing the response so that we can have DRY code?